hhs_oauth_server/urls.py

```diff
@@ -23,15 +21,11 @@
         name='openid_connect_userinfo'),
     url(r'^userinfo', userinfo_w_login,
```

Comment:
I already combined the via oauth and non-oauth flows for the rest of the API, are these two similar? Should this be removed to focus on the more used /connect/userinfo?